CondFormats/SiPixelObjects/interface/SiPixelVCal.h

```diff
@@ -0,0 +1,40 @@
+#ifndef SiPixelVCal_h
+#define SiPixelVCal_h
+#include <cstdint>
+#include <iostream>
+#include <map>
+#include <vector>
+#include "CondFormats/Serialization/interface/Serializable.h"
+
+class SiPixelVCal {
+public:
+  SiPixelVCal(){};
+  ~SiPixelVCal(){};
+
+  struct VCal {
+    float slope = 47.;
+    float offset = -60.;
+    COND_SERIALIZABLE;
+  };
+
+  inline void putSlopeAndOffset(std::map<unsigned int, VCal>& offset) { m_vcal = m_vcal; }
```

Comment:
The cause of the error is this line

Reply:
Thanks, it should be fixed now with the new commit https://github.com/cms-sw/cmssw/pull/29829/commits/fce246a99cb8bccf5f6f9f9ebe5070dbd99bcebf below.